II.pass-tests/10-paint-ford/paint-cars.js

```diff
@@ -0,0 +1,7 @@
+const paintShop = function (cars, newColour) {
+  // var paintedCars = cars.slice()
+  var paintedCars = JSON.parse(JSON.stringify(cars))// duplicating array without a refrence to original array
+  paintedCars[0].colour = newColour
```

Comment:
can we guarantee that the car to be painted will always be the first element?
How can we make this function more _reusable_? :)

Reply:
hmm I wrote the code based on test already written ... will discuss this with you in class